packages/ui/src/components/va-dropdown/VaDropdown.vue

```diff
@@ -85,9 +87,7 @@ export default defineComponent({
       set (val) { statefulVal.value = val },
     })
 
-    const computedClass = computed(() => ({
-      'va-dropdown--disabled': props.disabled,
-    }))
+    const computedClass = useBem('va-dropdown', pick(props, ['disabled']))
```

Comment:
How does this reactive? We likely losing reactivity here by passing props value, instead of reference. This must be computed or () => pick(

We also need to double check for this issue in other usages.